systemd/afterburn-checkin.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Descitpion=Afterburn Check In
+After=network-online.target
+
```

Comment:
We'll also want:

1. `After` greenboot is happy.
2. `ConditionKernelCommandLine=|ignition.platform.id=azure`

The first is harmless on distros that don't have greenboot.  Afterburn already reads `ignition.platform.id` so the second shouldn't further affect distro-independence.

Reply:
:+1: will add

Reply:
After speaking with @LorbusChris I put in:

```
After=boot-complete.target
```

Reply:
We can`After=boot-complete.target` (and probably `Requires=boot-complete.target` here (which is what greenboot does, when installed)
IIRC this unit is usually not activated - this will activate it however, and by default there is nothing blocking that target's activation (like a health-check runner a la greenboot)